projects/aca-shared/src/lib/services/app.service.ts

```diff
@@ -166,9 +166,11 @@ export class AppService implements OnDestroy {
       if (isReady) {
         this.loadRepositoryStatus();
         this.loadUserProfile();
-        setTimeout(() => {
-          this.openMobileAppDialog();
-        });
+        if (document.readyState === 'complete') {
```

Comment:
how that fixes the problem? if was not showing previously, before adding check for "readyState", it will not going to fire with this check too, as readyState might be loading... what does this check actually do to fix?

Reply:
basically I added timer in settimeout, if you want i can remove this check.